test/units/plugins/action/test_action.py

```diff
@@ -464,6 +464,7 @@ def build_module_command(env_string, shebang, cmd, arg_path=None, rm_tmp=None):
 
         mock_connection = MagicMock()
         mock_connection.build_module_command.side_effect = build_module_command
+        mock_connection._shell.get_remote_filename.return_value = 'amodule.py'
```

Comment:
This looks like the correct way to fix this.  However, since the task.action is set to "copy" above, maybe it would be more realistic to set this to "copy.py" here.